index.js

```diff
@@ -33,51 +44,107 @@ const Request = {
     const key = Verifier.manifestHash(manifest)
     const id = HypercoreID.normalize(key)
 
+    const isDrive = state.start !== state.end && c.uint8.decode(state) !== 0
```

Comment:
& 1 instead of !== 0 so it can support more flags in the future and also use c.uint instead of c.uint8